### ISSUE TITLE
Misses not counted in profiler?

### ISSUE THREAD

**Issue by drichmond** [bespoke-silicon-group/bsg_manycore#390]
@marcusnchow noticed that the profiler wasn't counting misses. I'm filing this proactively so it doesn't fall off our radar.

May have been fixed by #387? I'm not optimistic though.

**Comment by taylor-bsg** [comment]
Assign to Tommy

**Comment by drichmond** [comment]
@marcusnchow Can you post an example after you test with 4.3.0? Should be released later today. Ideally something Tommy can reproduce from a branch of bsg_manycore

**Comment by marcusnchow** [comment]
Yeah, an easy example you can use is the reduction benchmark in the baseline repo 
https://github.com/bespoke-silicon-group/baseline/tree/v4.1.0-stats/examples/reduction
I am using the [v4.1.0](https://github.com/bespoke-silicon-group/baseline/tree/v4.1.0-stats) branch of baseline and the new [v4.3.0](https://github.com/bespoke-silicon-group/bsg_bladerunner/commit/7e2b87cda4a51193c926a0690e6c5fb1fe09d5e5) bladerunner

You don't need to make any changes just run with 
```make default```

Once it's done you'll see a manycore_stats.log file where it reports no misses, but shows that there were bubbles due to misses. Like this...
```
Per-Tag Miss Stats
Miss Type                                        Misses            Accesses        Hit Rate (\%)
=================================================================================================
---------------------------------------------------------------------------------------------  Tag 0        
miss_icache                                           0               12421              100.00
miss_beq                                              0                 529              100.00
miss_bne                                              0                 350              100.00
miss_blt                                              0                 111              100.00
miss_bge                                              0                 388              100.00
miss_bltu                                             0                  88              100.00
miss_bgeu                                             0                  16              100.00
miss_jalr                                             0                 112              100.00
miss_total                                            0                1594              100.00
---------------------------------------------------------------------------------------------  Tag kernel    
miss_icache                                           0               14029              100.00
miss_beq                                              0                 584              100.00
miss_bne                                              0                 395              100.00
miss_blt                                              0                 120              100.00
miss_bge                                              0                 429              100.00
miss_bltu                                             0                  92              100.00
miss_bgeu                                             0                  16              100.00
miss_jalr                                             0                 128              100.00
miss_total                                            0                1764              100.00
=================================================================================================
Per-Tag Bubble Stats
Bubble Type                                      Cycles        \% of Bubbles   % of Total Cycles
=================================================================================================
---------------------------------------------------------------------------------------------  Tag 0  
bubble_branch_miss                                 1284               85.15                3.09
bubble_jalr_miss                                    224               14.85                0.54
bubble_icache_miss                                    0                0.00                0.00
bubble_total                                       1508              100.00                3.63
---------------------------------------------------------------------------------------------  Tag kernel   
bubble_branch_miss                                 1424               84.76                3.06
bubble_jalr_miss                                    256               15.24                0.55
bubble_icache_miss                                    0                0.00                0.00
bubble_total                                       1680              100.00                3.61
=================================================================================================
```

**Comment by drichmond** [comment]
@tommydcjung When will you have a chance to look at this?

**Comment by drichmond** [comment]
Fixed in #410
